board/fischertechnik/TXT/rootfs/bin/calibrate-touchscreen

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+calibrate() {
+	sudo TSLIB_TSDEVICE=/dev/input/event0 /usr/bin/ts_calibrate
+	/sbin/set-touchscreen-calibration-reset-flag
+}
+
+commit() {
+	/sbin/unset-touchscreen-calibration-reset-flag
+}
+
+reset() {
+	cp /media/sdcard/root/etc/pointercal /etc/pointercal
+	commit
```

Comment:
Now, this one is more of a style question, but in my opinion `commit` misleading here. Either use `rm` directly to show what is happening, or rename the `commit` (note: only the internal function, _not_ the command) function to something like `unlock` or `delete_flag`.